### PR TITLE
Task - Bulk Update Application Review endpoint

### PR DESCRIPTION
**Pull request by Khristophor**

### CODE REVIEW

File: app/Policies/ApplicationReviewPolicy.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+     * @return mixed
+     */
+    public function update(User $user, ApplicationReview $applicationReview)
+    {
+        $user->loadMissing('manager');
+        // TODO: Is there business logic around who can update/create application reviews?
+        // Does it need to be the manager who created the Job Poster, or can it be any manager/HR advisor?
+        // $applicationReview->loadMissing('job_application.job_poster.manager');
+        // return $user->isUpgradedManager() &&
+        //     $applicationReview->job_application->job_poster->manager->id === $user->manager->id;
```
Comment (Khristophor) [inline app/Policies/ApplicationReviewPolicy.php:28]: @tristan-orourke not sure if you know what the business logic is around creating/updating application reviews either

File: tests/Feature/Api/ApplicationReviewControllerTest.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+        $applicationReviews = ApplicationReview::whereIn('job_application_id', $applications->pluck('id')->all())->get();
+        foreach ($applicationReviews as $applicationReview) {
+            $applicationReview->fill(['review_status_id' => ReviewStatus::where('name', 'still_in')->first()->id]);
+        }
+
+        $response = $this->actingAs($manager->user)
+            ->postJson('api/v2/application-reviews/batch-update', $applicationReviews->toArray());
+        $response->assertOk();
+        $applicationReviews = $applicationReviews->makeHidden(['review_status', 'created_at', 'updated_at'])->toArray();
```
Comment (tristan-orourke) [inline tests/Feature/Api/ApplicationReviewControllerTest.php:37]: `makeHidden` - that's handy!

**Review by tristan-orourke — COMMENTED** [review]
Looks good, just need to adopt that policy logic from JobPolicy.

File: tests/Feature/Api/ApplicationReviewControllerTest.php (hunk 2 above)
Comment (yonikid15) [reply tests/Feature/Api/ApplicationReviewControllerTest.php:37]: Yea ^

**Review by yonikid15 — APPROVED** [review]
Nice Work 👍

**Pull request merged**